src/Candidates.js

```diff
@@ -0,0 +1,107 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+
+const Candidates = (props) => {
+  const { candidateList, handleVote } = props;
+
+  const list = candidateList
+    .sort((item1, item2) => {
+      if (item1.voteCount < item2.voteCount) {
+        return 1;
+      } else if (item1.voteCount > item2.voteCount) {
+        return -1;
+      } else return 0;
+    })
```

Comment:
```suggestion
   .sort(item1, item2) => return item1.voteCount - item2.voteCount;)
```
이렇게 한 줄로 줄여볼 수도 있을거 같아요!